cloud/ec2.go

```diff
@@ -721,6 +721,19 @@ func (m *ec2Manager) GetInstanceStatuses(ctx context.Context, hosts []host.Host)
 		for i := range hostsToCheck {
 			instance, ok := reservationsMap[*hostsToCheck[i]]
 			if !ok {
+				// Terminate an unknown host in the db
+				for _, h := range hosts {
+					if h.Id == *hostsToCheck[i] {
+						if err = h.Terminate(evergreen.User, "host is unknown to AWS"); err != nil {
+							grip.Error(message.WrapError(err, message.Fields{
```

Comment:
you can just call h.Terminate in the grip.Error(message.WrapError()) call to avoid the extra indentation.

Reply:
done